example/spec/setup/PopulateProvidersCollections.js

```diff
@@ -1,18 +1,86 @@
+'use strict';
 
 const { addProviders, addCollections } = require('@cumulus/integration-tests');
+const { s3 } = require('@cumulus/common/aws');
 const { loadConfig } = require('../helpers/testUtils');
+const fs = require('fs-extra');
 const config = loadConfig();
 
 const collectionsDirectory = './data/collections';
 const providersDirectory = './data/providers';
 
+/**
+ * Custom require to get the files from test data for upload to S3
+ *
+ * @param {Object} module - module
+ * @param {string} filename - filename
+ * @returns {undefined} - none
+ */
+function requireAsText(module, filename) {
+  module.exports = fs.readFileSync(filename, 'utf8'); // eslint-disable-line no-param-reassign
+}
+
+require.extensions['.PDR'] = requireAsText;
```

Comment:
From the node documentation for `require.extensions`: https://nodejs.org/api/modules.html#modules_require_extensions

> **Deprecated** In the past, this list has been used to load non-JavaScript modules into Node.js by compiling them on-demand. However, in practice, there are much better ways to do this, such as loading modules via some other Node.js program, or compiling them to JavaScript ahead of time.
> 
> Since the module system is locked, this feature will probably never go away. However, it may have subtle bugs and complexities that are best left untouched.